.env.sample

```diff
@@ -0,0 +1 @@
+UPPER_SNAKE_CASE=This is an example.
```

Comment:
UPPER_SNAKE_CASE or SECRET?

Reply:
@luongvo I want to give more detail on how the env variable looks like 🙏

Reply:
I meant, you seem to define a secret name as SECRET to display at https://github.com/nimblehq/flutter_templates/pull/10/files/45a1a333f4fb92355f529a72e4449c1b82d00e57

Reply:
@luongvo so you want to update `UPPER_SNAKE_CASE` to `SECRET` for consistency, right? 😄

Reply:
@manh-t not for consistency, but it should make to work, not just getting an non-exist config 💭

Reply:
@luongvo ah if so, I have added a guide to README to create `.env` file and add the required variables. This `.env.sample` file is an example. It doesn't make to work 🙏

Reply:
@manh-t we all know that env is a sample env, and what we should do after checking out this template is removing `.sample` extension and provide corresponding values to start the build, right? That's not the time for us to scan all credentials again to define, so I prefer to put `SECRET` var here or remove the sample code to get it out at https://github.com/nimblehq/flutter_templates/pull/10/files/45a1a333f4fb92355f529a72e4449c1b82d00e57 🙏

Reply:
@luongvo I understand what you mean. Let me update to `SECRET`. Thanks for the explanation 👍

Reply:
@luongvo updated 🚀